exporter/chronicleexporter/hostmetrics.go

```diff
@@ -85,13 +85,16 @@ func newHostMetricsReporter(cfg *Config, set component.TelemetrySettings, export
 
 func (hmr *hostMetricsReporter) start() {
 	ticker := time.NewTicker(5 * time.Minute)
```

Comment:
I think we can declare this in the go routine to make it more clear.

Reply:
Agreed